### ISSUE TITLE
[Bug]: TabList/Tab shows accessibility issues based on FastPass from Accessibility Insights for Web

### ISSUE THREAD

**Issue by dakotaJang** [microsoft/fluentui#30433]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System:
    OS: Linux 6.2 Ubuntu 22.04.2 LTS 22.04.2 LTS (Jammy Jellyfish)
    CPU: (16) x64 AMD EPYC 7763 64-Core Processor
    Memory: 56.56 GB / 62.78 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codesandbox.io/p/sandbox/currying-bird-7yygmr

### Bug Description

Using TabList/Tab shows accessibility issues based on FastPass from [Accessibility Insights for Web](https://accessibilityinsights.io/).
I used this [sandbox page](https://7yygmr.csb.app/) from [react.fluentui.dev docs](https://react.fluentui.dev/?path=/docs/components-tablist--default).

## Actual Behavior
Running FastPass from [Accessibility Insights for Web](https://accessibilityinsights.io/) on [sandbox page](https://7yygmr.csb.app/) shows following accessilibility issue:
- aria-hidden-focus: Ensures aria-hidden elements are not focusable nor contain focusable elements

## Expected Behavior
No accessibility issue when using TabList/Tab
- specifecially expecting no `aria-hidden-focus: Ensures aria-hidden elements are not focusable nor contain focusable elements`

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
This is caused by the `aria-hidden` utility elements created by tabster, and creates no practical accessibility issues.

We have this documented in our [known issues wiki](https://dev.azure.com/microsoftdesign/fluent-ui/_wiki/wikis/fluent-ui.wiki/334/Fluent-v9-known-issues?anchor=%60aria-hidden-focus%60-on-elements-with-%60data-tabster-dummy%60).
